scripts/Muon/GUI/Common/contexts/muon_data_context.py

```diff
@@ -193,6 +193,15 @@ def num_detectors(self):
             n_det = 1
         return n_det
 
+    @property
+    def num_points(self):
+        workspace_lengths = [len(self._loaded_data.get_data(run=run, instrument=self.instrument)['workspace']['OutputWorkspace'][0]
```

Comment:
can we split this line up, as it is a bit hard to read with all of the brackets

Reply:
Done

Reply:
Done